js/cards.js

```diff
@@ -85,15 +96,41 @@ export async function defineCardComponent() {
       return `${CLOUDINARY_URL}/q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
     }
 
+    resetCard() {
```

Comment:
The tabIndex is set to -1 so that if you try to use keyboard controls to flip the card, you don't have to tab twice (as it stops on both card-front and card-back, but I only want card-back to be stopped on. If you stop on card-front there's no action.

Reply:
Updated the name to `setupCardGacha`, but easily updated to something else if that makes more sense.